src/assets/branding/product.json

```
@@ -24,12 +24,12 @@
     "name": "CHE"
   },
   "docs": {
-    "devfile": "/docs/che-7/making-a-workspace-portable-using-a-devfile/",
-    "workspace": "/docs/che-7/workspaces-overview/",
-    "factory": "/docs/factories-getting-started.html",
-    "organization": "/docs/organizations.html",
-    "converting": "/docs/che-7/converting-a-che-6-workspace-to-a-che-7-devfile/",
-    "certificate": "/docs/che-7/installing-che-in-tls-mode-with-self-signed-certificates/",
-    "general": "/docs/che-7"
+    "devfile": "https://www.eclipse.org/che/docs/che-7/making-a-workspace-portable-using-a-devfile/",
+    "workspace": "https://www.eclipse.org/che/docs/che-7/workspaces-overview/",
+    "factory": "https://www.eclipse.org/che/docs/factories-getting-started.html",
+    "organization": "https://www.eclipse.org/che/docs/organizations.html",
+    "converting": "https://www.eclipse.org/che/docs/che-7/converting-a-che-6-workspace-to-a-che-7-devfile/",
+    "certificate": "https://www.eclipse.org/che/docs/che-7/installing-che-in-tls-mode-with-self-signed-certificates/",
```

Do you think it makes sense to change the link in the following way:
```suggestion
    "certificate": "https://www.eclipse.org/che/docs/che-7/installing-che-in-tls-mode-with-self-signed-certificates/#using-che-with-tls_installing-che-in-tls-mode-with-self-signed-certificates",
```

---

That make sense since this link is shown within the only particular error message.